### PR TITLE
Fixed IPython Displaying the Plasma Stratification (Table) even when the Logger is turned Off

### PR DESCRIPTION
**Pull request by DhruvSondhi**
This PR aims to fix a bug that had been introduced when we merged #1684. Currently, the Plasma Stratification table is displayed even when the logger is turned off. This needed to be fixed. I have added the `IPython App` logger for considering the log level status when configuring the logger. This fixes the bug.

**Description**

**Motivation and context**

**How has this been tested?**
- [x] Testing pipeline.
- [ ] Other. 

**Examples**

This is the Current bug.
image

This is the fixed version.
image

**Type of change**

- [x] Bug fix. 
- [ ] New feature. 
- [ ] Breaking change. 
- [ ] None of the above. 

**Checklist**

- [x] My change requires a change to the documentation.
    - [x] I have updated the documentation accordingly.
    - [x] (optional) I have built the documentation on my fork following [the instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html).
- [x] I have assigned and requested two reviewers for this pull request.

**Preview Available here:** https://dhruvsondhi.github.io/tardis/branch/fix_jupyter_display_log/io/optional/logging_configuration.html

### CODE REVIEW

File: tardis/io/logger/logger.py [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import logging
 import sys
-import logging
```
Comment (andrewfullard) [inline tardis/io/logger/logger.py:3]: Why is this changed?

File: tardis/io/logger/logger.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 import warnings
 import pyne.data
+
 from tardis.io.logger.colored_logger import ColoredFormatter, formatter_message
+from tardis.util.base import is_notebook
```
Comment (andrewfullard) [inline tardis/io/logger/logger.py:7]: Why is this added?

File: tardis/io/logger/logger.py (hunk 1 above)
Comment (DhruvSondhi) [reply tardis/io/logger/logger.py:3]: Duplicated import Logging

File: tardis/io/logger/logger.py (hunk 2 above)
Comment (DhruvSondhi) [reply tardis/io/logger/logger.py:7]: Wrong import. I will remove it. It is not being used in the current implementation for the `logging_state` function.

File: tardis/io/logger/logger.py [hunk 3]
```diff
@@ -1,8 +1,8 @@
 import logging
 import sys
-import logging
 import warnings
 import pyne.data
+
```
Comment (epassaro) [inline tardis/io/logger/logger.py:5]: why a newline?

**Review by epassaro — APPROVED** [review]

File: tardis/io/logger/logger.py (hunk 3 above)
Comment (DhruvSondhi) [reply tardis/io/logger/logger.py:5]: Just to demarcate the different levels of imports. Native library imports, imports from external modules, etc

**Review by andrewfullard — APPROVED** [review]
